### ISSUE TITLE
Accessibility: Content not included in landmarks - Main Content

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#440]
**Description**
All content on a webpage should be inside landmarks. This allows screen readers to understand the purpose of different areas of the page at a glance based on the defined roles that these landmarks have.

**Solution**
Add `role="main"` to  `<div id="main-container" class="container">`

**Comment by mhk33** [comment]
Fixed in dev/3.0 upgrade
